### ISSUE TITLE
Feature request: ingame command-line interface / game console

### ISSUE THREAD

**Issue by x-qq** [calref/cboe#235]
![sh](https://user-images.githubusercontent.com/55463047/72657501-beb6d500-39ad-11ea-92f4-c935cd7acb9e.png)

I feel like something a command-line interface (some form of text console) in the game could help in many ways, most obvious currently being access to various functionality that Linux users miss due to there being no menubar.

~ To show/hide, pgup/pgdn to scroll.

**Comment by wrldwzrd89** [comment]
Do you happen to know why Linux builds don't render a menu bar?

**Comment by x-qq** [comment]
It simply is not implemented. On Mac, menus are written using the mac-natie framework (Cocoa). On Windows, using windows-native menus (windows.h). On Linux the necessary functions are simply empty so there is no implementations.

There has been some discussions about trying to integrate wxwidgets or gtkmm menus into openboe, but nothing concrete so far.

**Comment by NQNStudios** [comment]
I don't think this is feasible. But we did add a menubar on Linux, right? Because someone reported a bug from it.
